core/models.go

```diff
@@ -496,6 +496,24 @@ func (s *Server) UnmarshalJSON(b []byte) error {
 	return nil
 }
 
+// IsValid returns true if server record is valid.
+func (s *Server) IsValid() bool {
+	return s.Name != "" &&
+		s.Station != "" &&
+		s.Hostname != "" &&
+		len(s.Technologies) > 0 &&
+		len(s.Locations) > 0
+}
+
+func (s *Servers) Validate() error {
```

Comment:
nitpick: my understanding of `Validate` is that if some data set isn't quite right then it gets fixed, by undertaking some specific actions.
I don't think this is the case here, right?
Is `IsValid` used in any other contexes?

Maybe then simply remove the `IsValid`, snap the check in-place in the method over the range, and rename it from `Validate` to `IsValid`?

Reply:
Also, it would be nice to add a comment also to the `Validate` - just to keep coherent way of commenting what's the purpose of a given method.